be/src/storage/vectorized/delta_writer.cpp

```diff
@@ -185,12 +179,11 @@ Status DeltaWriter::_init() {
     _tablet_schema = writer_context.tablet_schema;
     _reset_mem_table();
     _flush_token = _storage_engine->memtable_flush_executor()->create_flush_token();
-    _set_state(kWriting);
+    _set_state(kPrepared);
```

Comment:
kPrepared -> kInitialized

Reply:
done, change in https://github.com/StarRocks/starrocks/pull/4333/commits/0edd9627577ca337779715829abc0f9228517099